client/v2/algod/rawTransaction.go

```diff
@@ -28,7 +28,7 @@ func (s *SendRawTransaction) Do(ctx context.Context, headers ...*common.Header)
 		}
 	}
 	if addContentType {
-		headers = append(headers, &common.Header{"Content-Type", "application/x-binary"})
+		headers = append(headers, &common.Header{Key: "Content-Type", Value: "application/x-binary"})
```

Comment:
Is the linter requiring always using field names in struct contruction? 
There are certain situation when I explicitly want to _not_ use field names. For example, if I'm worried that another field might get added in the future and want the already defined structs to cause a compile error.

Reply:
Yes, the linter currently enforces this. We can choose to exclude this rule from the check in the golanci yaml. 

I don't feel very strongly about it, but it could be nice to have backwards incompatible changes caught as a side effect. I'm not sure if the current codebase has meaningful places that would catch this though?

Reply:
Is there a `//lint: ignore` directive that could be applied here?

Reply:
There is a directive for golangci, but none for `go vet` (it looks like go vet cannot be disabled for a particular line). We can run `go vet -composites=false ./...` to disable the rule altogether